tests/search/struct_and_map_types/paged_attribute_fields/test.sd

```diff
@@ -0,0 +1,229 @@
+# Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+search test {
+  document test {
+    struct elem {
+      field name type string {}
+      field weight type int {}
+    }
+    struct complex_elem {
+      field name type string {}
+      field weight type int {}
+      # This field is not searchable
+      field str_map type map<string, int> {}
+    }
+    field elem_array type array<elem> {
+      # All struct-fields are attributes
+      indexing: summary
+      struct-field name {
+        indexing: attribute
+        attribute {
+          fast-search
+          paged
+        }
+      }
+      struct-field weight {
+        indexing: attribute
+        attribute: paged
+      }
+    }
+    field elem_map type map<string, elem> {
+      # All struct-fields are attributes
+      indexing: summary
+      struct-field key {
+        indexing: attribute
+        attribute {
+          fast-search
+          paged
+        }
+      }
+      struct-field value.name {
+        indexing: attribute
+        attribute {
+          fast-search
+          paged
+        }
+      }
+      struct-field value.weight {
+        indexing: attribute
+        attribute: paged
+      }
+    }
+    field elem_map_2 type map<string, elem> {
+      # Some struct-fields are attributes
+      indexing: summary
+      struct-field key {
+        indexing: attribute
+        attribute {
```

Comment:
Consider using _attribute: paged_ as done else where in the file when only one parameter is specified.